mscore/editstyle.cpp

```diff
@@ -1405,5 +1433,19 @@ void EditStyle::resetTextStyle(Pid pid)
       textStyleChanged(textStyles->currentRow());     // update GUI
       cs->update();
       }
+
+void EditStyle::renameUserStyle() {
+      int row = textStyles->currentRow();
+      Tid tid = Tid(textStyles->item(row)->data(Qt::UserRole).toInt());
+      int idx = int(tid) - int(Tid::USER1);
```

Comment:
Same as above, we need to make sure `idx` is in the [0,5] range to avoid crashes.